Modules/CalcOffence-3_0.lua

```diff
@@ -1172,13 +1178,76 @@ function calcs.offence(env, actor, activeSkill)
+
+		-- Seismic Cry only Exerts Slam Skills
+		output.SeismicHitEffect = 1
+		if activeSkill.skillTypes[SkillType.SlamSkill] then
+			local numSeismicExerts = env.modDB:Sum("BASE", nil, "NumSeismicExerts") or 0
+			if numSeismicExerts > 0 and not skillFlags.warcry then
+				local SeismicHitMultiplier = skillModList:Sum("BASE", cfg, "SeismicHitMultiplier") / 100
+				for index, value in ipairs(actor.activeSkillList) do
+					if value.activeEffect.gemData.name == "Seismic Cry" then
+						-- recursively calculate the values for Seismic Cry
+						-- only actual Cooldown and WarcryCastTime are returned
+						local seismicCryData = calcs.offence(env, actor, actor.activeSkillList[index], true)
+
+						output.SeismicCryCooldown = seismicCryData.Cooldown
+						output.SeismicCryCastTime = seismicCryData.WarcryCastTime
+						break
+					end
+				end
+				output.SeismicDmgImpact = 0
+				for i = 1, numSeismicExerts do
+					output.SeismicDmgImpact = output.SeismicDmgImpact + (i * SeismicHitMultiplier)
+				end
+				output.SeismicAvgDmg = output.SeismicDmgImpact / numSeismicExerts
+				-- calculate ratio of uptime versus downtime (including opportunity cost of casting the warcry)
+				output.SeismicUpTimeRatio = m_min((numSeismicExerts / output.Speed) / (output.SeismicCryCooldown + output.SeismicCryCastTime), 1.0)
+				exertedUptime = m_max(exertedUptime, output.SeismicUpTimeRatio)
+				output.SeismicHitEffect = 1 + (output.SeismicAvgDmg * output.SeismicUpTimeRatio)
+			end
+		end
+
+		-- Intimidating Cry Exerts Attacks
+		output.IntimidatingHitEffect = 1
+		if activeSkill.skillTypes[SkillType.Attack] then
+			local numIntimidatingExerts = env.modDB:Sum("BASE", nil, "NumIntimidatingExerts") or 0
+			if numIntimidatingExerts > 0 and not skillFlags.warcry then
+				for index, value in ipairs(actor.activeSkillList) do
+					if value.activeEffect.gemData.name == "Intimidating Cry" then
+						-- recursively calculate the values for Intimidating Cry
+						-- only actual Cooldown and WarcryCastTime are returned
+						local intimidatingCryData = calcs.offence(env, actor, actor.activeSkillList[index], true)
+
+						output.IntimidatingCryCooldown = intimidatingCryData.Cooldown
+						output.IntimidatingCryCastTime = intimidatingCryData.WarcryCastTime
+						break
+					end
+				end
+				-- calculate ratio of uptime versus downtime
+				output.IntimidatingUpTimeRatio = m_min((numIntimidatingExerts / output.Speed) / (output.IntimidatingCryCooldown + output.IntimidatingCryCastTime), 1.0)
+				exertedUptime = m_max(exertedUptime, output.IntimidatingUpTimeRatio)
+				-- intimidating cry guarantees double damage for its attacks; therefore, its hit effect
+				-- is calculated as the improvement over the non-intimidated double damage chance
+				output.IntimidatingHitEffect = 1 + (1 - output.DoubleDamageChance / 100) * output.IntimidatingUpTimeRatio
+			end
+		end
+
+		-- Account for INC and MORE increases for Exerted Attacks
+		if exertedUptime > 0 then
```

Comment:
I think this section is actually never getting done? I can't see either of these mods in the mod breakdowns in the Calcs tab, even with the build you provided.

Regardless, the More Damage from Seismic Cry should not have ModFlag.Attack. The lines say "Exerted Attacks deal x% More Damage", indicating generic more damage and not more attack damage. Confirmed by Mark here: https://www.reddit.com/r/pathofexile/comments/hb56kp/path_of_exile_harvest_gem_information/fv75ooy/?context=2

Reply:
![Proof](https://user-images.githubusercontent.com/1735956/86985330-04837780-c15f-11ea-9aee-04cd8040309d.png)